app/Controller/SongsController.php

```diff
@@ -194,8 +194,8 @@ public function albums() {
                 'Song' => array(
                     'fields' => array('Song.id', 'Song.band', 'Song.album', 'Song.cover'),
                     'conditions' => $subQuery,
-                    'order' => 'Song.album',
-                    'limit' => 36
+                    'order' => ['Song.band', 'Song.album'],
+                    'limit' => 60
```

Comment:
I'll try to reproduce it tonight and make a more thorough test.

As for the sorting, as a temporary workaround you can put only 'Song.band', also have you tried using `array()` instead of `[]`. I'm not sure PHP handles arrays the same when using these two syntax.